geoips/errors.py

```diff
@@ -4,6 +4,27 @@
 """GeoIPS error module."""
 
 
+class NoValidFilesError(Exception):
+    """Exception to be raised when there are no valid files found.
+
+    Can be used from reader plugins if no valid files were passed to the reader.
+    """
+
+    pass
+
+
+class OutputFormatterDatelineError(Exception):
+    """Exception to be raised when there is a dateline error in an output formatter."""
```

Comment:
Also these docstrings are not much more informative than the class name... do we need more info here?

Reply:
https://github.com/NRLMMD-GEOIPS/geoips/issues/845

Reply:
^ Opened an issue, resolving